src/Sfa.Tl.ResultsAndCertification.Web/Controllers/AdminDashboardController.cs

```diff
@@ -437,9 +438,24 @@ public async Task<IActionResult> AdminReviewChangesIndustryPlacementAsync(AdminR
             {
                 return View(model);
             }
+            model.LoggedInUser = $"{this.User.FindFirstValue(ClaimTypes.GivenName)} {this.User.FindFirstValue(ClaimTypes.Surname)}";
```

Comment:
It cannot be seen here but I noticed in this method that we read from the cache before checking the model is valid. We don't need to call the cache if the model is invalid, so the cache has to be read after the check

Reply:
moved this